lib/app/routes/core.rb

```diff
@@ -143,6 +147,7 @@ def planned_languages
         def tracks
           ExercismWeb::Presenters::Tracks
```

Comment:
Yepp, you would need to loop through the way the current `active`, `upcoming`, and `planned` methods are doing right in-place where you need it.